### DISCUSSION TITLE
[??] Debian11 + свой лист

### DISCUSSION THREAD

**Discussion by mrdiega** [bol-van/zapret#719] [Q&A]
Всем привет. Подскажите, как установить zapret на Debian 11 и сделать так, чтобы zapret работал только на сайты из моего списка, без влияния на всю сеть?
Хочу сделать как в этом форке для винды https://github.com/Flowseal/zapret-discord-youtube ([list-general.txt](https://github.com/Flowseal/zapret-discord-youtube/blob/main/list-general.txt) + [ipset-discord.txt](https://github.com/Flowseal/zapret-discord-youtube/blob/main/ipset-discord.txt))
Желательно простыми словами) Боюсь всё поломать 😁

**Comment by HentayHentay** [comment]
Это будет сложно и больно если Вы не обладаете навыками линукса на среднем уровне

**Reply by mrdiega** [reply]
да я справлюсь, если подробно объяснит кто-то 😌

**Reply by HentayHentay** [reply]
Ок
Походу поиск отвалился ,но помогу
[тут про дебиан](https://github.com/bol-van/zapret/blob/master/docs/manual_setup.txt)

**Comment by bol-van** [comment]
docs/quick_start
manual_setup не нужен
